tracks/preprocessing/track_preprocessing.py

```diff
@@ -0,0 +1,108 @@
+import argparse
```

Comment:
I am not checking this file, as discussed, the final PR should have the pre-processed json file instead. As already mentioned, please stick to camelCase naming convention (also for function names) to be consistent with the rest of the code.